code-challenges.js

```diff
@@ -31,24 +36,63 @@ const temp = 42
 // const temp = 212
 // Expected output: "212 is at boiling point"
 
+if (temp < 212) {
+    console.log(`${temp}° is below the boiling point!`);
+  } else if (temp === 212) {
+    console.log(`${temp}° is at the boiling point!`);
+  } else if (temp > 212) {
+    console.log(`${temp}° is above the boiling point!`);
+
+// END RESULT:
+//All good after running node in the terminal. Expect output generated.
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+// In order to create the code that will combine the two arrays and return the length using the test variables provided below I will need to utilize the .concat() method to merge the two arrays. 
+//By doing this the original array(s) is unchanged and just combined. 
+//I will use the "let" command to establish each array to designate the Padres 1984, and Padres 1988 World Series Runs, and then use the "console.log" command in conjuction with the .concat command followed by the .length. 
+//Afterwards I will use "comboArrays" to establish the variable for the combined arrays.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+let padresTotalRuns = padres1984WorldSeriesRuns.concat(
+    padres1998WorldSeriesRuns
+  ).length;
+  console.log(padresTotalRuns)
+
+// END RESULT:
+//All good after running node in the terminal. Expect output generated.
+
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
+// In order to create the code that will reverse the letters of a string using the test variable providied below, I will need to utilize the .reverse() method following my the currentCohort variable.
 
 const currentCohort = "Foxtrot 2022"
 // Expected output: "2202 tortxoF"
 
+//console.log(currentCohort.reverse())
+
+
+// END RESULT:
+// When I ran node in the terminal it rendered that .reverse() is not a function, I'm not sure as to why after troubleshooting.
+// I posted the issue I was experiecing in Slack and a few classmates and an instructor were about to help me troubleshoot further, here is the code that eventually became successful for me:
```

Comment:
I really like that you are adding this process in here.  It is super helpful to see your whole process and how you get to the working code.  Great work!